examples/workflow/authoring/src/human-interaction.ts

```diff
@@ -139,6 +139,10 @@ async function promptForApproval(approver: string, workflowClient: DaprWorkflowC
   }
 }
 
+process.on('SIGTERM', () => {
```

Comment:
![image](https://github.com/user-attachments/assets/983f61d9-0f5b-4a8e-868a-4154086cf166)
https://github.com/grpc/grpc-node/issues/2233

Proper fix would be adding error handler to take care of CANCELLED status. Although it is not critical for tutorials but for general javascript SDK client, it should be handled

Reply:
will take a look.  thank you for this

Reply:
Agreed, it would need a better handling through some error handler kind of mechanism.